actions/utils.ts

```diff
@@ -359,3 +365,103 @@ export function isComposableCowCompatible(code: string): boolean {
     return code.includes(sighash.slice(2));
   });
 }
+
+type ParsedError = {
+  errorNameOrSelector?: string;
+  message?: string;
+}
+
+/**
+ * Given a raw ABI-encoded custom error returned from a revert, extract the selector and optionally a message.
+ * @param abi of the custom error, which may or may not be parameterised.
+ * @returns an empty parsed error if assumptions don't hold, otherwise the selector and message if applicable.
+ */
+const rawErrorDecode = (abi: string): ParsedError  => {
+  if (abi.length == 10) {
```

Comment:
Triple the fun:

```suggestion
  if (abi.length === 10) {
```

Reply:
Yes, it's really ABI then. When passing back a custom error as a reversion message in solidity, it returns the custom error selector, which is formatted exactly the same way as when making a function call (formatting the ABI).

So for example, when calling `deposit` in the `WETH9` contract, the actual function you call is just `deposit()`, there is no parameter as the amount being passed in is in `msg.value`. So, in this case, the ABI is actually only `0xd0e30db0`.

Same case for custom errors that don't have any parameters. :+1: